### DISCUSSION TITLE
No viable replacement for 'render at native resolution' setting

### DISCUSSION THREAD

**Discussion by coaltars** [ppy/osu#30609] [Q&A]
I'm trying to convert from Stable to Lazer, but I play with strange settings; I run at 1280x1024 resolution with Render at native resolution enabled. I've tried multiple times to replicate the behavior of this on Lazer, with Screen scaling and/or resolution, yet I have not found a way.

OpenTabletDriver settings:
![gF7Pfyv](https://github.com/user-attachments/assets/94c65e8f-8a69-4875-b050-9d297a64a004)

osu!stable settings:
![Pzrghso](https://github.com/user-attachments/assets/83500b8b-e526-4f11-a29d-6ad6ac56dae0)

I'm not sure if there's something I'm missing, but if there isn't, maybe there is a possibility this feature could be added to Lazer. Thanks

**Comment by peppy** [comment]
It sounds like you want to set screen scaling to `Everything` and then adjust the horizontal % to 33.3% and vertical % to 94.8%.

**Reply by coaltars** [reply]
This results in an extremely vertically-dominant screen, which is not what I was aiming for; my goal is to run the client window at 1920x1080, while playing with an effective resolution of 1280x1024. Sorry if this doesn't make total sense.

**Reply by peppy** [reply]
I was basing those numbers on you running the game at 3840x1080 as per your screenshot. If that's not the case, you'll need to adjust. I'm sure you can trial and error to get the size you want?
